managed-identity-storage-blob/README.md

```diff
@@ -0,0 +1,53 @@
+# Access Storage Blob with managed identity
+
+This sample shows how to access Storage Blob with managed identity in `Azure Spring Cloud`.
+
+## Prerequisite
+
+* 
+* [JDK 8](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [Maven 3.0 and above](http://maven.apache.org/install.html)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+* An existing Storage Account. If you need to create a Storage Account , you can use the Azure Portal or [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest).
```

Comment:
Let's put a link to the official storage account docs: https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal